core/src/main/java/software/amazon/awssdk/metrics/AwsSdkMetrics.java

```diff
@@ -295,10 +295,10 @@
                             } else if (HOST_METRIC_NAME.equals(key)) {
                                 hostMetricName = value;
                             } else {
-                                LogFactory.getLog(AwsSdkMetrics.class).debug("Ignoring unrecognized parameter: " + part);
+                                LoggerFactory.getLogger(AwsSdkMetrics.class).debug("Ignoring unrecognized parameter: " + part);
```

Comment:
Use existing log variable here and other places in this class.
Also remove string addition